vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/SpreadsheetDropdownFactory.java

```diff
@@ -56,13 +56,18 @@ public SpreadsheetDropdownFactory toColIndex(int i) {
   }
 
   @Override
-  public Component getCustomComponentForCell(Cell cell, int rowIndex, int columnIndex, Spreadsheet spreadsheet,
-      Sheet sheet) {
+  public Component getCustomComponentForCell(Cell cell, int rowIndex, int columnIndex,
+      Spreadsheet spreadsheet, Sheet sheet) {
     if (spreadsheet.getActiveSheetIndex() == 0
-        && rowIndex >= fromRowIndex && rowIndex <= toRowIndex && columnIndex >= fromColIndex && columnIndex <= toColIndex) {
-      if(cell==null || cell.getStringCellValue().isEmpty()) {
-        return initCustomComboBox(rowIndex, columnIndex,
-            spreadsheet);
+        && rowIndex >= fromRowIndex && rowIndex <= toRowIndex && columnIndex >= fromColIndex
+        && columnIndex <= toColIndex) {
```

Comment:
good point, some of this is already part of the next PR, since a refactoring of the DropdownFactory was necessary to create multiple different columns with dropdowns.